cli/spcli/createminer/create_miner.go

```diff
@@ -0,0 +1,130 @@
+package createminer
```

Comment:
why does this get moved? just a shuffle, or is this useful as an import elsewhere? if so, should it be moved out of `cli` entirely?

Reply:
We are situating spcli to be the CLI for lotus (the binary), instead of folding it into Curio (lotus-miner style).
As Curio plans to support new SPs who have not used lotus-miner before, we need spcli to make new miner IDs. 

Since spcli is not connected to Curio nor lotus-miner nor Boost, we consider it Lotus-specific. 
Who will actually manage it is yet to be determined, but having a tool that provides all the chain-only SP functionalities improves predictabilitiy.

Reply:
that's fine, I'm just wondering if we have what's looking like a library rather than a cli whether we should make a new package for it. if this is the only piece so far then that's OK. If you end up having to have imports elsewhere that have `cli` in them then it suggests these pieces shouldn't remain in here but we need a new top-level package .. or a package elsewhere